space-concordia-robotics/robotics-prototype
# Teensyduino libs install 362

# Assignee Section

## Description
The CMake script in `robot/rover` installs all needed Teensyduino libraries, using `install_libraries.sh` in `robot/rover`. By default, Arduino libraries are installed in `~/Arduino/libraries`, so this script won't work if someone changed the default installation directories of the Arduino IDE.

### Steps for Testing
1. Delete all libraries in `~/Arduino/libraries` (except for unrelated libraries installed for personal use)
2. Go to `robotics-prototype/robot/rover` 
3. `mkdir build && cd build`
4. `cmake ..` and ensure that the library installations are logged in the terminal. 
5. `make` 

closes #362 

The approval from all software team leads is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.


For Pull Requests that do not include code changes, it is not required to perform the tests above.
